### DISCUSSION TITLE
Online only config option for a wildloader.

### DISCUSSION THREAD

**Discussion by Shybella** [BG-Software-LLC/WildLoaders#60] [Pending Feature Requests]
Hello, 

Is it possible to add a feature for chunk loading only while the player who placed the wildloaders is online. If the player goes offline the chunks that the player loaded with wildloaders will be unloaded from the world until they login again. 

(sidenote)
I am using your plugin on Arclight ( https://github.com/IzzelAliz/Arclight ) This is a Forge + Spigot hybrid and it works great, not sure if this was intentional but compatibility is naturally good. 

Thank you.

**Comment by James-P-Bennett** [comment]
Bumping, as this has become a much needed option for larger servers
